gemma-core/src/main/java/ubic/gemma/core/analysis/singleCell/aggregate/SingleCellExpressionExperimentAggregatorServiceImpl.java

```diff
@@ -260,6 +268,9 @@ public QuantitationType aggregateVectors( ExpressionExperiment ee, QuantitationT
                 }
             }
         }
+        if ( config.getMask() != null ) {
+            details.append( "\n" ).append( " Mask: " ).append( config.getMask() );
+        }
```

Comment:
It would be nice to put some statistics such as the fraction of cells that were masked.